docker-compose.yaml.template

```diff
@@ -13,6 +13,7 @@ services:
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock"
       - save:/home/cnb
+      - /home/saveu/agentLogs:/home/cnb/agentLogs/
```

Comment:
Do you see, how the value from `application.properties` leads to inconsistency, as discussed in https://github.com/cqfn/save-cloud/pull/108#discussion_r634480183?

Reply:
Fixed